test/ios/getProjectConfig.spec.js

```diff
@@ -1,31 +1,25 @@
-const path = require('path');
-const expect = require('chai').expect;
+jest.autoMockOff();
+
 const getProjectConfig = require('../../src/config/ios').projectConfig;
 const mockFs = require('mock-fs');
 const projects = require('../fixtures/projects');
 
 describe('ios::getProjectConfig', () => {
+  const userConfig = {};
 
-  before(() => {
-    mockFs({ testDir: projects });
-  });
+  beforeEach(() => mockFs({ testDir: projects }));
```

Comment:
That was false positive. We need these hooks. As I already mentioned, that was the case because 
```js
typeof null === object
```
Now this behavior is fixed.